image_width: 1280
image_height: 960
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [521.305668, 0, 632.190486, 0, 518.581771, 437.368475, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.224668, 0.031387, 0.002094, 0.001785, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [315.864044, 0, 642.374059, 0, 0, 311.883514, 441.329364, 0, 0, 0, 1, 0]